apps/talk_signaling.sh

```diff
@@ -0,0 +1,403 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
+
+# shellcheck disable=2034,2059
+true
+# shellcheck source=lib.sh
+NC_UPDATE=1 && TURN_INSTALL=1 . <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
```

Comment:
BTW: I don't like it that you have to enter the domain at the very beginning of the script... maybe we can do it like in onlyoffice-docker?

Reply:
Sure, please change.

Reply:
@szaimen  Could you please add this change to the new PR?